azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/AbstractFunctionMojo.java

```diff
@@ -323,6 +354,8 @@ public Map<String, String> getTelemetryProperties() {
         result.put(DISABLE_APP_INSIGHTS_KEY, String.valueOf(isDisableAppInsights()));
         final boolean isDeployToFunctionSlot = getDeploymentSlotSetting() != null && StringUtils.isNotEmpty(getDeploymentSlotSetting().getName());
         result.put(FUNCTION_DEPLOY_TO_SLOT_KEY, String.valueOf(isDeployToFunctionSlot));
+        result.put(FUNCTION_IS_NATIVE_EXECUTABLE, String.valueOf(isNativeExecutable()));
+        result.put(FUNCTION_CUSTOM_HANDLER_ARGS, customHandlerArgs);
```

Comment:
Will ‘customHandlerArgs’ contains any PII data like file location? If so, we should not record them in telemetry but we could track whether users set custom handler args here

Reply:
I prefer to keep line 357, so that we could track how many users use native executable, for custom handler args, I'm not sure whether we need track it, so may be could remove it